### ISSUE TITLE
Fonts served via packr fail beginning at v1.15.0

### ISSUE THREAD

**Issue by glimberg** [gobuffalo/packr#105]
Beginning with v1.15.0 of packr, boxed webfonts give the following errors in Chrome when loaded:

```
Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.woff2
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.woff
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.woff
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.ttf
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.ttf
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.woff
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.ttf
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.woff2
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.woff
a0cbf4b62a8fe084:1 Failed to decode downloaded font: http://localtest.me:8000/fonts/glyphicons-halflings-regular.ttf
10OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
a0cbf4b62a8fe084:1 OTS parsing error: invalid version tag
```

Any attempt to use the glyphicons in the above fonts results in the lovely unicode box

**Comment by markbates** [comment]
Can you provide a simple reproducible app please? Thanks.

**Comment by mycroftsk** [comment]
I have same problem on Windows 10 and 7, works fine in Manjaro Linux. In file.go
 ```
// file.go
func (s *virtualFile) Read(p []byte) (int, error) {
	return s.buf.Read(p)
}

s.buf.Read(p) return EOF error and last 512 bytes are thrown away.

**Comment by markbates** [comment]
If anybody has a patch, I would very much welcome it.

**Comment by markbates** [comment]
This should be fixed with `1.20.0`.

**Comment by david-kroell** [comment]
I am using windows and packr `1.21.0` and fonts served from Buffalo do not work.

I also had this issue with earlier versions and thus updated `the version in my `mod.go`
```
github.com/gobuffalo/packr v1.21.0
```
I haven't tried this on linux.
Any thoughts?
